MastodonSDK/Sources/MastodonUI/View/Content/StatusView.swift

```diff
@@ -517,9 +517,10 @@ extension StatusView.Style {
         // link preview card
         statusView.contentContainer.addArrangedSubview(statusView.statusCardControl)
 
-        statusView.spoilerOverlayView.translatesAutoresizingMaskIntoConstraints = false
-        statusView.containerStackView.addSubview(statusView.spoilerOverlayView)
-        statusView.contentContainer.pinTo(to: statusView.spoilerOverlayView)
+        statusView.containerStackView.addArrangedSubview(statusView.spoilerOverlayView)
+        NSLayoutConstraint.activate([
+            statusView.spoilerOverlayView.heightAnchor.constraint(equalToConstant: 128).priority(.defaultHigh)
```

Comment:
Magic Number yay. Can also be smaller.

Reply:
If I have a short post with a CW, the CW is bigger than the post. Not a blocker to :)

![Simulator Screen Recording - iPhone 15 Pro - 2024-05-22 at 16 11 45](https://github.com/mastodon/mastodon-ios/assets/2580019/80d4e78b-447c-4dcf-aabd-c22bc51a6282)

Reply:
Yes, that's because currently it's one-size-fits-them-all, not sure how we should handle such cases, I'd keep it as is and refine with design eventually.